helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterModelProvider.java

```diff
@@ -154,39 +329,35 @@ public static ClusterModel generateClusterModelFromCurrentState(
       for (String resourceName : replicaMap.keySet()) {
         Set<AssignableReplica> replicas = replicaMap.get(resourceName);
```

Comment:
Use Map.Entry to iterate over this replicaMap? For minor performance gain?

Reply:
Sure.